app/src/main/java/programatorus/client/screens/debugger/messagelist/MessageListItem.kt

```diff
@@ -0,0 +1,11 @@
+package programatorus.client.screens.debugger.messagelist
+
+data class MessageListItem(
+    val message: String,
+    val isLocal: Boolean
+) {
+    companion object {
+        fun local(message: String) = MessageListItem(message, true)
+        fun remote(message: String) = MessageListItem(message, false)
+    }
+}
```

Comment:
`sealed class/interface` would be better suited here. But you can leave it.

Reply:
Leaving as it is